frame/election-provider-multi-phase/src/benchmarking.rs

```diff
@@ -254,6 +255,69 @@ frame_benchmarking::benchmarks! {
 		assert!(<MultiPhase<T>>::queued_solution().is_some());
 	}
 
+	#[extra]
+	trim_assignments_length {
+		// number of votes in snapshot.
+		let v in (T::BenchmarkingConfig::VOTERS[0]) .. T::BenchmarkingConfig::VOTERS[1];
+		// number of targets in snapshot.
+		let t in (T::BenchmarkingConfig::TARGETS[0]) .. T::BenchmarkingConfig::TARGETS[1];
+		// number of assignments, i.e. compact.len(). This means the active nominators, thus must be
+		// a subset of `v` component.
+		let a in (T::BenchmarkingConfig::ACTIVE_VOTERS[0]) .. T::BenchmarkingConfig::ACTIVE_VOTERS[1];
+		// number of desired targets. Must be a subset of `t` component.
+		let d in (T::BenchmarkingConfig::DESIRED_TARGETS[0]) .. T::BenchmarkingConfig::DESIRED_TARGETS[1];
```

Comment:
I'd be curious to try these with some very large numbers on steroids as well, as `BenchmarkingConfig` is more about a sane config I think.

Reply:
@kianenigma you can override the limits in the CLI using `--low` and `--high`: https://github.com/paritytech/substrate/blob/master/utils/frame/benchmarking-cli/src/lib.rs#L46